### ISSUE TITLE
[BUG] integrationcli integrations create delivers a wall of JSON

### ISSUE THREAD

**Issue by DinoChiesa** [GoogleCloudPlatform/application-integration-management-toolkit#330]
## Summary
when I use `integrationcli integrations create` to upload? import? a .json file containing an integration definition, the result of the command is a wall of JSON as large as my file.  It can be 1000 or more lines. This is not what users want from an interactive command line tool.  They want to see a "it worked" or "it didn't work" message.    

## Actual Experience

```
$ integrationcli integrations create -f ./apigee-pdu-check-bg4a11.json  -n apigee-pdu-check-1
{
        "name": "projects/14182523368/locations/us-west1/integrations/integration-name-1/versions/e1a44e8a-8f52-45f5-ae61-e4a6351ec247",
        "description": "description here.",
        "origin": "UI",
        "snapshotNumber": "1",
        "updateTime": "2024-12-20T01:27:30.722962Z",
        "lockHolder": "person@example.com",
        "createTime": "2024-12-20T01:27:30.722962Z",
        "lastModifierEmail": "person@example.com",
        "state": "DRAFT",
        "triggerConfigs": [
                {
                        "label": "Query Organization",
                        "startTasks": [
                                {
                                        "taskId": "2"
                                }
                        ],
      ....over 1000 more lines here....
```

## Version
```
$ integrationcli --version
integrationcli version 0.78.0 date: 2024-12-10T17:57:38Z [commit: 9371c4e]
```

## Expected behavior 
Some smaller, human-scale response that indicates success or failure.

**Comment by srinandan** [comment]
As with issue #328, the CLI's behavior matches that of the API. It is also not a bug, rather an enhancement.

One of the uses of `integrationcli` is as you say, to be used interactively and such uses don't require the entire response JSON. However, `integrationcli` is frequently used in automation (like Github Actions) and those use cases (when combined with tools like `jq`) do require access to the full response JSON (because they are likely need to access fields from the response like snapshot number). Returning a non-JSON output like "it worked" breaks automation use cases.

Proposal: Introduce a new flag (`--concise`) to omit triggerConfigs, taskConfigs, integrationParameters, errorCatcherConfigs, integrationConfigParameters. The rest of the fields are static and perhaps useful to see. The default behavior will continue.

**Comment by srinandan** [comment]
Please take a look at release [v0.79.0](https://github.com/GoogleCloudPlatform/application-integration-management-toolkit/releases/tag/v0.79.0), a number of suggestions from your issue have been incorporated.

**Comment by DinoChiesa** [comment]
Can I set `--concise` to be on by default ?

**Comment by ssvaidyanathan** [comment]
how about we use that as part of the `prefs` command

**Comment by srinandan** [comment]
Yes, I can add a preference that provides basic output. But I don't want to change the default since that is a breaking change.
